### DISCUSSION TITLE
Insurgency: Sandstorm server fails to launch, except in debug mode

### DISCUSSION THREAD

**Discussion by poperigby** [GameServerManagers/LinuxGSM#4072] [Q&A]
When I try to launch the Insurgeny: Sandstorm server, it fails:
```
[ FAIL ] Starting inssserver: Unable to start Haddock Server
```

I checked `log/console/inssserver-console.log`, and it contained this:
```
fish: No matches for wildcard 'Canyon?Scenario=Scenario_Crossing_Checkpoint_Security?MaxPlayers=28?game=CheckpointHardcore'. See `help wildcards-globbing`.
   ./InsurgencyServer-Linux-Shipping Canyon?Scenario=Scenario_Crossing_Checkpoint_Security?MaxPlayers=28?game=CheckpointHardcore -Port=27102 -QueryPort=27131 -hostname='Haddock Server' -Rcon -RconPassword= -RconListenPort=27015 -log -GSLTToken=REDACTED
                                     ^
```

I thought it must be a problem with `fish` (my shell), so I tried running it with `bash`. That didn't work either, and `inssserver-console.log` was empty.

The thing is, it works fine if I launch it with `./inssserver debug`.

OS: Arch Linux

**Comment by kvnxiao** [comment]
Getting the same issue, any support/findings for this?

**Reply by poperigby** [reply]
Unfortunately no :confused:

**Reply by kvnxiao** [reply]
I found the fix for it. I was using `zsh` and also got an issue with running the command in the tmux session, similar to your `fish` shell error -- something related to "no matches found".
What I did was simply add `SHELL=/usr/bin/sh` to the env before running the command
and it works!
